src/test/resources/data/InstructorFeedbackEditPageUiTest.json

```diff
@@ -321,7 +321,6 @@
       "resultsVisibleFromTime": "2026-05-01T21:59:00Z",
       "timeZone": "Australia/Victoria",
       "gracePeriod": 10,
-      "feedbackSessionType": "STANDARD",
```

Comment:
What is this doing?

Reply:
`feedbackSessionType` attribute is removed as data migration completed. There is no point to keep the attribute in test JSON data.